groupease-server/src/main/java/io/github/groupease/channel/JpaChannelDao.java

```diff
@@ -107,6 +111,10 @@ public Channel create(
 
         entityManager.persist(toCreate);
 
+        /* Refresh instance. */
+        entityManager.flush();
+        entityManager.refresh(toCreate);
```

Comment:
@Groupease/cs6675_spring_2018 - I found situations where after a query, the instance was not being refreshed to contain the new lastUpdatedOn timestamp and other data.  These two calls fixed it.